### ISSUE TITLE
Load offscreen questions

### ISSUE THREAD

**Issue by ojchase** [ojchase/OKCupidQuestionFilterExtension#2]
Precondition - the user is on a question page. Either Agree, Disagree, or Find Out. For the filter the user is clicking, at least one question has been answered and could be shown.
Action - the user clicks a filter

Expected Behavior: At least one question that matches the filter is shown
Actual Behavior: It depends entirely on what questions have been previously loaded. Sometimes zero are shown.

This is because this currently works by taking an existing page and removing elements from it. The extension does not try to interact with OKCupid's servers. We do not have the full set of question/answer data unless we scroll all the way to the end.

Basically, we need to load more of the page automatically. The "scroll" event triggers more of the page to load. We need to send some.

I am undecided whether we should try to load the entire page of results (some users have a lot!) or just one more screen at a time. If one screen at a time, should we loop until one question is found, or a full set (OKCupid seems to do 10 on its own)?

**Comment by ojchase** [comment]
Autoscrolling tab I had up that I won't pursue now: https://stackoverflow.com/questions/11715646/scroll-automatically-to-the-bottom-of-the-page
